### ISSUE TITLE
Time limit of AlevinQC

### ISSUE THREAD

**Issue by nictru** [nf-core/scrnaseq#334]
### Description of the bug

Currently, the time limit of AlevinQC is hard coded to 120h [here](https://github.com/nf-core/scrnaseq/blob/c5a64459fec1506304f7064639f3f8e12d971269/conf/modules.config#L187) since #196 was solved via #197. In my case, this led to Slurm PartitionTimeLimit problems (basically nothing was executed because the time limit of the queue is lower than 120h). Usually one can solve this by setting the `max_time` parameter. However, this parameter does not affect the AlevinQC time limit.

As an experienced nextflow user, one can easily find a temporary fix through adding this to `nextflow.config`:
```
process {
    withName: 'ALEVINQC' {
        time = '5.h'
    }
}
```
But I think figuring this out might be hard for users with less experience.

Some ideas for a permanent fix:

- Use the `check_max` function for setting the limit
- Assign one of the standard process labels (maybe `process_low` + `process_long`) and tell users to perform a manual override of the time limit if they really hit the time limit

I can also open a PR myself, but maybe I am missing something here, so let's discuss first

### System information

Slurm
`nf-core/scrnaseq` version 2.6.0

**Comment by grst** [comment]
Hi, 

this should definitely use the check_max function. PR would be welcome!

**Comment by nictru** [comment]
Great, will open one later today!

**Comment by grst** [comment]
Closed by #335
